lib/websocket_webmks_uint8utf8.rb

```diff
@@ -0,0 +1,31 @@
+class WebsocketWebmksUint8utf8 < WebsocketSSLSocket
```

Comment:
Wouldn't be better to inherit from the `WebsocketWebmks`?

Reply:
It would BUT then we can't invoke `WebsocketSSLSocket`'s constructor to setup SSL stuff for us. On the other hand, we must not invoke `WebsocketWebmks`'s constructor because it'll start the wrong driver for us, so...

Can you feel the pain? 😄 

In other words:

```
WebsocketWebmksUint8utf8 <--- WebsocketWebmks <--- WebsocketSSLSocket
            |------------- super.super ------------------^
```

From what I know, Ruby does not support invoking `super.super` methods, therefore we need to do the ugly copy-pasting here... But maybe I missed something?